lib/providers/firebase-rest.js

```diff
@@ -0,0 +1,54 @@
+import config from 'lib/config'
+
+const {FIREBASE_DATABASE_URL} = config
+
+// Helpers to make it easier to work with the Radio4000 Firebase database.
+
+// 5 second timeout:
+
+async function fetchAndParse(url) {
+	const host = FIREBASE_DATABASE_URL
+
+	// stop the request if hanging
+	const controller = new AbortController()
```

Comment:
🪄 til

Reply:
yeah.... but seems more client side in the end